contract/pkg/bucket/messages.go

```diff
@@ -29,6 +29,13 @@ const (
 	OFFLINE
 )
 
+var NodeTags = map[string]byte{
```

Comment:
Tag bring misleading that one node can have two of them in parallel. 
What if we use State?

Reply:
Updated